### ISSUE TITLE
Inline frame missing a text alternative

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#648]
Every inline frame (iframe) should have a text alternative that summarizes its content or purpose.

Siteimprove report: https://my2.siteimprove.com/Accessibility/7333/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&pageSegments=&ruleName=sia-r13&ruleId=13&issueKind=1&exceptTags=1,2&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2

Accessibility guidelines for `<iframe>` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#accessibility
